ddp-workspace/projects/ddp-dsm-ui/src/app/FON/pages/activities/components/activity/activity.component.scss

```diff
@@ -364,13 +364,6 @@ $field-disabled-border: $__Mulled_wine_transparent;
           .ddp-answer-field {
             padding: unset !important;
 
-            /* TODO: remove this temp FIX!! just to fix the wrong config form back-end */
```

Comment:
Can you please remove similar TODO with a hardcode on line 423 ?

Reply:
Sure, It's done